contracts/BTRUST.sol

```diff
@@ -1,11 +1,300 @@
-pragma solidity >=0.5.8 <0.6.0;
+pragma solidity ^0.5.16;
 
-import "@openzeppelin/contracts/token/ERC20/ERC20.sol";

```

Comment:
compound token also has `pragma experimental ABIEncoderV2;` - may not actually be used but let's just add this to ours too to keep aligned